pycbc/inference/sampler.py

```diff
@@ -181,8 +276,14 @@ def run(self, niterations, **kwargs):
             The list of log proposal densities for the walkers at positions p,
             with shape (nwalkers, ndim).
         """
-        return self._sampler.run_mcmc(niterations, **kwargs)
+        try:
+            p0 = kwargs.pop('p0')
```

Comment:
If ``p0`` is required why not put it in the docstring and as an option? This kinds of hides that its required.